### PR TITLE
Clap v4

### PR DESCRIPTION
**Pull request by chevdor**
- Update to clap 4
- update deps
- fix string formatting

### CODE REVIEW

File: src/bin/kickstart.rs [hunk 1]
```diff
@@ -2,52 +2,53 @@ use std::env;
...
 use kickstart::validate::validate_file;
 
-pub fn build_cli() -> App<'static, 'static> {
-    App::new("kickstart")
-        .version(crate_version!())
-        .author(crate_authors!())
-        .about(crate_description!())
-        .setting(AppSettings::SubcommandsNegateReqs)
+pub fn build_cli() -> Command {
+    command!()
```
Comment (Keats) [inline src/bin/kickstart.rs:12]: Can we switch to using the struct way instead of the macro?

**Review by Keats — CHANGES_REQUESTED** [review]

File: src/bin/kickstart.rs (hunk 1 above)
Comment (chevdor) [reply src/bin/kickstart.rs:12]: Sure, there are also a few other issues I am addressing.

**Comment by chevdor** [comment]
I am getting an error about the template.toml being missing whereas it is not, not sure yet what is going on there.

**Comment by Keats** [comment]
The error for the CI is probably just needing to bump the msrv

**Comment by chevdor** [comment]
I am seeing usage errors as well. So I think this is not 100% ready. I need to dig since I don't get the error.
I am fetching and providing a valid template.toml but the app complains about the file not existing.

**Comment by Keats** [comment]
I'm doing it in https://github.com/Keats/kickstart/pull/62

**Pull request closed without merging**